### PR TITLE
Add pao engineers to designated owner files

### PR DESCRIPTION
**Pull request by yanirq**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,9 +1,12 @@
 approvers:
   - jmencak
   - kpouget
+  - MarSik
 reviewers:
   - ArangoGutierrez
   - jmencak
   - dagrayvid
   - kpouget
+  - MarSik
```
Comment (jmencak) [inline OWNERS:10]: We can have more people as reviewers from the CNF team here if you wish.

**Review by jmencak — COMMENTED** [review]
This mostly looks good to me, with the exception of:
- 4 users do not seem to be me members of the openshift org.  Can we have them added first?
- The location of some `OWNERS` files is not ideal in my view.  One example is the `test` directory.  The `test` directory is mostly NTO e2e tests, meaning PAO team is supposed to review and also approve them.  Not sure what the right strategy is, but there are a couple of options:  1) restructuring of the directory itself, 2) moving the `OWNERS` file to the `performanceprofile` folder itself, 3) keeping things simple and having one top level `OWNERS` file with only the approvers as suggested by this PR and adding the PAO reviewers.  So looking at this PR, I wonder if 3) in the long run would be the best option since we are trying to merge those operators into one anyway?

File: assets/performanceprofile/OWNERS [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+# default (and sane) config is to inherited from
+# parent OWNERS, see:
+# https://www.kubernetes.dev/docs/guide/owners/#owners
+
+approvers:
+  - fromanirh
```
Comment (fromanirh) [inline assets/performanceprofile/OWNERS:8]: considering the shared maintainership, I'll start lean with 2 (two) people here

File: assets/performanceprofile/OWNERS [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  - swatisehgal
+  - yanirq
+  - Tal-or
+  - jlojosnegros  
+reviewers:
```
Comment (fromanirh) [inline assets/performanceprofile/OWNERS:13]: and here

File: examples/OWNERS [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+# default (and sane) config is to inherited from
+# parent OWNERS, see:
+# https://www.kubernetes.dev/docs/guide/owners/#owners
+
+approvers:
```
Comment (fromanirh) [inline examples/OWNERS:7]: same for all others

**Comment by yanirq** [comment]
> 4 users do not seem to be me members of the openshift org. Can we have them added first?
In progress.

> 2) moving the OWNERS file to the performanceprofile folder itself, 

I was thinking about that but kept in mind that we might need reviews on other folders like util for example.
I can move the OWNERS to performanceprofile for now

> 3) in the long run would be the best option since we are trying to merge those operators into one anyway?
That was my initial thought before we decided to switch to this scheme

@jmencak  We can remove all the OWNERS files in the sub folders, add all the CNF reviews to the root file and add 3 CNF approvers (Martin, Francesco and myself). That would simplify the structure and get the rest of the engineers more familiar with the repo.

**Comment by jmencak** [comment]
> @jmencak We can remove all the OWNERS files in the sub folders, add all the CNF reviewers to the root file and add 3 CNF approvers (Martin, Francesco and myself). That would simplify the structure and get the rest of the engineers more familiar with the repo.

Yes, it does seem to be a better option looking at this PR now.  You seem to be pushing for 3 approvers from your side even though I originally wanted to keep the number of approvers low.  OK, let's open it up and add those 3 engineers from your side as approvers and also @dagrayvid from our side.

**Comment by jmencak** [comment]
Also, @ArangoGutierrez please let us know if you want to be removed from the `OWNERS` file as you don't seem to be involved with the project any longer.

**Comment by dagrayvid** [comment]
Over all this looks good, but I do wonder if having 6 approvers is too many for one project of this size?

I think there is a benefit to having a small number of people who are aware of every PR that is merged. With too many approvers, it increases the number of PRs that each individual approver is unaware of. The potential negatives of having more approvers can probably be mitigated so long as we are committed to thorough reviews.

Perhaps you (@jmencak, @yanirq) disagree? I would like to know the justification for the large number of approvers before we merge this PR.

**Comment by jmencak** [comment]
> Over all this looks good, but I do wonder if having 6 approvers is too many for one project of this size?
> 
> I think there is a benefit to having a small number of people who are aware of every PR that is merged. With too many approvers, it increases the number of PRs that each individual approver is unaware of. The potential negatives of having more approvers can probably be mitigated so long as we are committed to thorough reviews.
> 
> Perhaps you (@jmencak, @yanirq) disagree? I would like to know the justification for the large number of approvers before we merge this PR.

Since the start of the project, with very few exceptions, I was the single person issuing `/approve` even during my PTO.  This no longer scales with PAO merged.  So adding approvers will help people on PTO not block important functionality.  Where it doesn't help is potentially getting stuff in that people leading the project do not approve of during their absence.  This is a risk I'm willing to take.

Now, is 6 people a lot?  If it is, we can decrease this number from NTO side to two.  As you, David, are now more familiar with codebase than Kevin, I'm sure Kevin would not be offended to be removed as an approver.  On the other hand, I *know* Kevin would not approve something I would not approve of.  It is a matter of trust.  So I'm completely happy to have 3 NTO approvers and I do not see the risk there.  Now, from PAO side, I do not see a problem with a few engineers being approvers too.  I'd like those engineers suggested by @msivak or @fromanirh though.

I believe that `/approve` will be conscientiously issued by people from NTO/PAO teams mostly based on the type of a PR.  If the majority of the code will affect NTO, I expect NTO people to approve and vice versa.  If you agree, let's try to keep this open a bit more and deal with the situation as problems arise.  Fun fact, looking at k8s, there are 1-37 approvers in their `OWNERS` files.

**Comment by MarSik** [comment]
>  I'd like those engineers suggested by @msivak or @fromanirh though

Well it would indeed be @fromanirh as the most experienced engineer on the PAO side and @yanirq who has worked with you for the past year on making this merge happen (so he is trusted from our side indeed).

My specific permissions aren't critical, I have other stuff I need to handle so I use my approves sparingly, mostly for the architecture PRs. However PAO is not a tiny project anymore and for reasons of vacations and sick leaves, I prefer to have the power just in case a bug fix needs to get in.

**Comment by dagrayvid** [comment]
Thanks @jmencak and @MarSik for the explanations.

> I believe that /approve will be conscientiously issued by people from NTO/PAO teams mostly based on the type of a PR. If the majority of the code will affect NTO, I expect NTO people to approve and vice versa. If you agree, let's try to keep this open a bit more and deal with the situation as problems arise.

With this in mind, I think having 6 approvers is fine.

/lgtm

**Comment by jmencak** [comment]
I think I misinterpreted the `/verify-owners` command :/.  Let's test all.

/test all

**Comment by jmencak** [comment]
Also, @ArangoGutierrez , could you [comment on preference to receive review requests for this project](https://github.com/openshift/cluster-node-tuning-operator/pull/335#issuecomment-1088425076)?  We are happy to have you, but it would be suboptimal if you do not wish to participate.

**Comment by kpouget** [comment]
> I'm sure Kevin would not be offended to be removed as an approver.

indeed I wouldn't be offended, I think it makes sense to _replace_ me by @dagrayvid given his implication on the project

**Comment by ArangoGutierrez** [comment]
> Also, @ArangoGutierrez , could you [comment on preference to receive review requests for this project](https://github.com/openshift/cluster-node-tuning-operator/pull/335#issuecomment-1088425076)? We are happy to have you, but it would be suboptimal if you do not wish to participate.

It's ok, the project now has an entire team behind it. I agree with the new OWNERS layout

**Comment by jmencak** [comment]
> It's ok, the project now has an entire team behind it. I agree with the new OWNERS layout

I'm afraid it is not clear from your response what is ok.  Do you want to work as an active project reviewer or not?  If not, then there is no reason for the bot to pick you as one.  So, I'm asking, do you prefer to be dropped or not.

**Comment by ArangoGutierrez** [comment]
> > It's ok, the project now has an entire team behind it. I agree with the new OWNERS layout
> 
> I'm afraid it is not clear from your response what is ok. Do you want to work as an active project reviewer or not? If not, then there is no reason for the bot to pick you as one. So, I'm asking, do you prefer to be dropped or not.

remove `ArangoGutierrez` from NTO, thanks. 
Hope that is clear enough for you

**Comment by jmencak** [comment]
There's trailing whitespace on 3 lines of the `OWNERS` file, but we can fix that later.
/approve

**Pull request merged**